### PR TITLE
Correct default options

### PR DESCRIPTION
**Pull request by yodas7**
The default options were different from the description, so I fixed them.
They need to be fixed before being passed to the user.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -26,19 +26,18 @@ function createPerTypeTokenMap(
   }, {}) as Partial<Record<UnifiedTokenType, Record<string, string>>>;
 }
 
+export type Options = {
+  prefix?: string | undefined;
+  cssVarPrefix?: string | undefined;
+};
```
Comment (rinjInTokyo) [inline src/index.ts:32]:
```
export type Options = {
  prefix?: string;
  cssVarPrefix?: string;
};
```
The undefined definition maybe redundant?

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ module.exports = {
   variants: {
     extend: {},
   },
-  plugins: [daikinPlugin(), ...otherPlugins],
+  plugins: [daikinPlugin({}), ...otherPlugins],
```
Comment (rinjInTokyo) [inline README.md:25]: If just use `daikinPlugin()`, are there any problems happening here?

File: src/index.ts (hunk 1 above)
Comment (yodas7) [reply src/index.ts:32]: Actually it depends on user's tsconfig.
If user enables `exactOptionalPropertyTypes`, user cannot do `options.prefix = undefined` though it's allowed in our plugin.
see https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes.

File: README.md (hunk 2 above)
Comment (yodas7) [reply README.md:25]: Tailwind plugin factory seems not to support making params optional, so following error will occur.

スクリーンショット 2024-12-20 14 23 31

**Review by poetrainy — APPROVED** [review]

**Review by rinjInTokyo — APPROVED** [review]

**Pull request merged**